### PR TITLE
Updated README and exception handling and environmental variables and database update function

### PR DESCRIPTION
**Pull request by nathan-weinberg**
fixes poor README (I can't read) and exception handling and Dan's poor shell scripts and admittedly a small error I **might** have made

### CODE REVIEW

File: src/timbot.py [hunk 1]
```diff
@@ -173,12 +172,7 @@ def main():
                                        database=os.environ['SQL_DATABASE'])
 
     except Exception as err:
-        if err.errno == errorcode.ER_ACCESS_DENIED_ERROR:
-            print("Something is wrong with your user name or password")
-        elif err.errno == errorcode.ER_BAD_DB_ERROR:
-            print("Database does not exist")
-        else:
-            print(err)
+        print(err)
```
Comment (timflannagan) [inline src/timbot.py:184]: Great work Nathan!

**Comment by nathan-weinberg** [comment]
@Daniel-Pivonka approval button is near the top

**Comment by nathan-weinberg** [comment]
@timflannagan1 see my previous comment

**Comment by nathan-weinberg** [comment]
@adk3798 please merge this **crucial** hotfix

File: src/timbot.py (hunk 1 above)
Comment (nathan-weinberg) [reply src/timbot.py:184]: Thanks Tim!

File: scripts/run-timbot-local.sh [hunk 2]
```diff
@@ -11,6 +11,18 @@ elif [[ -z "$SLACK_CHANNEL_NAME" ]]; then
...
+    exit 1
+elif [[ -z "$SQL_USER" ]]; then
+    echo "You need to set the \$SQL_USER"
+    exit 1
+elif [[ -z "$SQL_PASSWORD" ]]; then
+    echo "You need to set the \$SQL_PASSWORD"
+    exit 1
+elif [[ -z "$SQL_DATABASE" ]]; then
+    echo "You need to set the \$SQL_DATABASE"
+    exit 1
```
Comment (timflannagan) [inline scripts/run-timbot-local.sh:25]: :puke:

**Review by timflannagan — APPROVED** [review]

**Pull request merged**